packages/backend/src/apps/lettersg/auth/is-still-verified.ts

```diff
@@ -0,0 +1,16 @@
+import type { IGlobalVariable } from '@plumber/types'
+
+import { verifyApiKey } from './verify-api-key'
+
+export default async function isStillVerified(
+  $: IGlobalVariable,
+): Promise<boolean> {
+  const authData = $.auth.data
+
+  if (!authData || !authData.apiKey) {
```

Comment:
nit: consider the simpler Typescript syntax `!authData?.apiKey` to improve readability